payments/swish/other-features.md

```diff
@@ -191,37 +191,320 @@ the nature of the problem.
 The problem name and description will often help narrow down the specifics
 of the problem.
 
-### Error types from Swish and third parties
+## Swish API errors
 
 All Swish error types will have the following URI in front of type:
-`{{ page.api_url }}/psp/<errordetail>/swish`
+`{{ page.api_url }}/psp/swish/payments/<errortype>`
```

Comment:
Won't URLs like these make it hard to some day eventually perform redirects to the documentation about the specific problems? `/psp/errordetail/swish` can be created as a resource without any authentication much easier than a resource beneath `/psp/swish/payments/`, no?

Reply:
Looks like I've got fooled by my localhost version of the project. When I trigger errors on internaltest environment, the errortype looks like this:
`https://api.internaltest.payex.com/psp/errordetail/<errortype>`.
I'll update the PR with correct content 👍